File: iOS/Part2.CH06/Library Buddy/Book.swift
//
//  Book.swift
//  Library Buddy
//
//  Created by Shaun Lewis on 6/27/19.
//  Copyright © 2019 Shaun Lewis. All rights reserved.
//

import Foundation

struct Book: Codable {
    let title: String
    let authors: [String]
    let isbn: String
    let pageCount: Int
    let fiction: Bool
}

File: Android/kotlin/app/src/main/java/org/oreilly/nmd/Book.kt
package org.oreilly.nmd

data class Book(
    var title: String,
    var authors: MutableList<String>,
    var isbn: String,
    var pageCount: Int,
    var isFiction: Boolean
) {

    constructor() : this("", mutableListOf<String>(), "", 0, false) {
        // no op
    }

}
